src/js/components/Socials/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const Socials = ({ socials }) => {
+
+  const socialsList = socials =>
```

Comment:
This function will be created executed on each component render. Maybe it makes sense to make this component a class? like:
```jsx
class Socials extends {
  _renderSocial = item => (...);

   render() {
       return (
             ...
             {socials.map(this._renderSocial)}
        );
   };
}
```